aten/src/ATen/test/vulkan_api_test.cpp

```diff
@@ -430,11 +430,11 @@ TEST(VulkanAPITest, conv2d) {
         height,
       };
     }
-  } weights {83, input.channels, 13, 2};
+  } weights {1, input.channels, 3, 3};
 
-  const auto input_cpu = at::randn(input.size(), at::device(at::kCPU).dtype(at::kFloat));
-  const auto weights_cpu = at::randn(weights.size(), at::device(at::kCPU).dtype(at::kFloat));
-  const auto bias_cpu = at::randn({weights.output_channels}, at::device(at::kCPU).dtype(at::kFloat));
+  const auto input_cpu = at::randn(input.size(), at::device(at::kCPU).dtype(at::kFloat))*5;
+  const auto weights_cpu = at::randn(weights.size(), at::device(at::kCPU).dtype(at::kFloat))*5;
+  const auto bias_cpu = at::randn({weights.output_channels}, at::device(at::kCPU).dtype(at::kFloat))*5;
```

Comment:
why *5, testing ?

Reply:
ah, I added these during testing and forgot to remove them, same for the one below. Good eye!